frontend/src/pages/customer-history/libs/options/options.ts

```diff
@@ -0,0 +1,10 @@
+const options = [
+  { label: 'Any', value: '' },
+  { label: 'Done', value: 'done' },
+  { label: 'Pending', value: 'pending' },
+  { label: 'Canceled', value: 'canceled' },
+  { label: 'Confirmed', value: 'confirmed' },
+  { label: 'Picking up', value: 'picking_up' },
+];
```

Comment:
use OrderStatus enum for value
also rename to const orderStatusOptions
and add return type